os/drivers/testcase/kernel_testcase_drv.c

```diff
@@ -120,6 +133,36 @@ static int kernel_test_drv_ioctl(FAR struct file *filep, int cmd, unsigned long
 		break;
 #endif
 
+#ifdef CONFIG_EXAMPLES_MPU_TEST
+	case TESTIOC_MPUTEST:
+		ret = OK;
+		struct mputest_arg_s *obj = (struct mputest_arg_s*)arg;
+		
+		if (!obj) {
+			return -EINVAL;
+		}
+
+		if (obj->type == MPUTEST_KERNEL_CODE) {
```

Comment:
For future work, it had better use switch-case instead of if-else.
If there's no change like that, please ignore this comment.